schema.graphql

```diff
@@ -129,6 +129,37 @@ type Investment @entity {
   datetime: Date!
 }
 
+type PortfolioJson @jsonField {
+  did: String!
+  number: Int!
+}
+
+type Leg @jsonField {
+  ticker: String! @index
+  amount: String!
+  from: PortfolioJson
+  to: PortfolioJson
```

Comment:
there is also a `direction` field in tooling-gql. Are we dropping it?

Reply:
Yeah, tooling-gql would calculate it after querying the DB, but now we don't have any code that happens during the query.

A caller should be able to figure it out with `direction: did === from.did ? 'Outgoing' : 'Incoming'`.

If you have any ideas for keeping I am open to hearing them.